packages/core/click-handler.js

```diff
@@ -127,12 +116,12 @@ async function onClick(event) {
   }
 }
 
-export default function(_pluginManager) {
+export default function(_matches) {
   $body.undelegate(LINK_SELECTOR, 'click', onClick);
   $body.undelegate(LINK_SELECTOR, 'mouseup', onClick);
 
   $body.delegate(LINK_SELECTOR, 'click', onClick);
   $body.delegate(LINK_SELECTOR, 'mouseup', onClick);
 
-  pluginManager = _pluginManager;
+  matches = _matches;
```

Comment:
For simplicity I tweaked the `click-handler` according to my needs. As mentioned above, it will go away soon.